22-discovery.ks

```diff
@@ -31,45 +37,83 @@ echo " * enabling required system services"
 systemctl enable ipmi.service
 systemctl enable foreman-proxy.service
 systemctl enable discovery-fetch-extensions.path
+systemctl enable discovery-menu.service
+
+# register service is started manually from discovery-menu
+systemctl disable discovery-register.service
+
+echo " * disabling some unused system services"
+systemctl disable ipmi.service
```

Comment:
We do but the service is not involved AFAIK. @stbenjam can you test this when doing the ipmi patch please?